src/test/controller/BaseController.ts

```diff
@@ -1,21 +1,55 @@
 import express, { Express } from 'express';
 import { App } from '../../main/App';
 import redisMock, { RedisClient } from 'redis-mock';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const firebasemock = require('firebase-mock');
```

Comment:
Unfortunately seems like `firebase-mock` doesn't have types for typescript ): hmm let's see if we can find a mock that comes with type definitions